lib/rules/selector-max-type/__tests__/index.js

```diff
@@ -549,3 +549,82 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  config: [0, { ignore: ["next-sibling"] }],
+  skipBasicChecks: true,
+  syntax: "scss",
+
+  accept: [
+    {
+      code: ".foo + a {}"
+    },
+    {
+      code: ".foo, .bar + a {}"
+    },
+    {
+      code: ".foo + a + b {}"
+    },
+    {
+      code: ".foo { .bar + a {} }"
```

Comment:
Let's use standard syntax here by adding the _nesting selector_:

```suggestion
      code: ".foo { & .bar + a {} }"
```